backend/danswer/db/connector_credential_pair.py

```diff
@@ -72,6 +72,7 @@ def get_last_successful_attempt_time(
     credential_id: int,
     embedding_model: EmbeddingModel,
     db_session: Session,
+    earliest_index: float,
```

Comment:
Slight nit: the most useful/frequently changed args first, least frequently changed (or most predictable or unimportant for the reader) last. db_session is always the same so throw it at the end.

Caller should generally try to follow same arg order as the function def so it's less likely for someone to miss something if not args are required etc.